refinery/analysis_manager/schemas/AnalysisConfig.json

```diff
@@ -0,0 +1,92 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "oneOf": [
+    {
+      "required": [
+        "custom_name",
+        "studyUuid",
+        "toolUuid",
+        "user_id",
+        "workflowUuid"
```

Comment:
@mccalluc So this was dependent on how the old ways of launching analyses configs were structured. I didn't want to change too much this first time around so I left the structure as it was